roles/create_cluster/defaults/main.yml

```diff
@@ -20,8 +20,25 @@ HTTP_PROXY: ""
 HTTPS_PROXY: ""
 NO_PROXY: ""
 
-manifest_templates:
-  - 50-worker-nm-fix-ipv6.yml
-  - 50-worker-remove-ipi-leftovers.yml
-  - 02-fix-ingress-config.yml
-  - 01-master-node-scheduler.yml
+create_cluster__single_node_openshift_enabled: "{{ single_node_openshift_enabled | default(false) }}"
+
+# Accepted values for "High Availability Mode"
+# 1. 'None' (when `single_node_openshift_enabled` is True)
+#     Requires one master node and no worker nodes -- SNO.
+# 2. 'Full' (default)
+#     Requires at least three master nodes for High Availability.
+create_cluster__high_availability_mode: "{{ create_cluster__single_node_openshift_enabled | ternary('None', 'Full') }}"
```

Comment:
The Title case strings are counter-intuitive given (almost?) all the others are lowercase, or case insensitive. Make these lowercase and use a `| title` filter in the create cluster payload creation.

Reply:
Addressed